packages/preview/supercharged-dhbw/1.6.0/lib.typ

```diff
@@ -0,0 +1,328 @@
+#import "@preview/codelst:2.0.1": *
+#import "acronym-lib.typ": init-acronyms, print-acronyms, acr, acrpl, acrs, acrspl, acrl, acrlpl, acrf, acrfpl
+#import "titlepage.typ": *
+#import "confidentiality-statement.typ": *
+#import "declaration-of-authorship.typ": *
+#import "check-attributes.typ": *
+
+// Workaround for the lack of an `std` scope.
+#let std-bibliography = bibliography
+
+#let supercharged-dhbw(
+  title: none,
+  authors: (),
+  language: none,
+  at-university: none,
```

Comment:
Renaming this argument technically means that this update is a breaking change, so if you follow semantic versioning, this version should be `2.0.0` and not `1.6.0` (but following semantic versioning is not really a requirement for packages even if the version numbers looks like it, so it is up to you to decide if you actually want to do it or not).

Reply:
Hey @elegaanz,

good suggestion, thanks 🙏🏻

I changed the version to 2.0.0